config/checkstyle_checks.xml

```diff
@@ -137,6 +137,13 @@
     <property name="message"
               value="Old site links should not be used, please use https://checkstyle.org"/>
   </module>
+  <module name="RegexpSingleline">
+    <property name="id" value="noDefaultCommentWithOtherVisibility"/>
+    <property name="format" value="/\* default \*/ (private|protected|public)"/>
```

Comment:
Regular expression between this and PMD should be similar, like allowance for extra spaces.